src/utils/get-error-pcc.js

```diff
@@ -3,7 +3,7 @@ export default (string) => {
     return null;
   }
 
-  const [, pcc = null] = string.match(/NO AGREEMENT EXISTS FOR AGENCY - ([A-Z0-9]+)/) || [];
+  const [, pcc = null] = string.match(/NO AGREEMENT EXISTS FOR AGENCY(?:\s+)-(?:\s+)([A-Z0-9]+)/) || [];
```

Comment:
Could be simplified to `\s+-\s+`.
Please check and update if it works.

Also I would like to see PCC limited to 3-4 symbols:
`([A-Z0-9]{3,4})`